### PR TITLE
docs: clarify plugins are auto-registered, remove components default

### LINKED ISSUE #9295: docs: plugins/ directory are auto-registered? why
### Environment

no

### Reproduction

All plugins in your plugins/ directory are auto-registered, so you should not add them to your nuxt.config separately.
But in [Nuxt Configuration Reference](https://nuxt.com/docs/api/configuration/nuxt-config#nuxt-configuration-reference),  all demo is in the `plugins` directory.

----

image

https://nuxt.com/docs/api/configuration/nuxt-config#plugins-1

### Describe the bug

documentation is unclear

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by BetonZM**
### 🔗 Linked issue

resolves #9295

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Added link to plugins guide page where users can find more information about plugins.
- Also removed blank default value at css category

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/schema/src/config/app.ts [hunk 1]
```diff
@@ -165,6 +165,11 @@ export default defineUntypedSchema({
    *
    * It can also be an object with `src` and `mode` keys.
    *
+   * @note Plugins are also auto-registered from plugins/ directory and do not need to be listed. All plugins are deduplicated  by their src path.
+   *
+   * @see https://nuxt.com/docs/guide/directory-structure/plugins
+   *
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:171]:
```suggestion
   * @note Plugins are also auto-registered from plugins/ directory and do not need to be listed. All plugins are deduplicated by their src path.
   *
   * @see https://nuxt.com/docs/guide/directory-structure/plugins
```

File: packages/schema/src/config/app.ts [hunk 2]
```diff
@@ -165,6 +165,10 @@ export default defineUntypedSchema({
    *
    * It can also be an object with `src` and `mode` keys.
    *
+   * @note Plugins are also auto-registered from plugins/ directory and do not need to be listed. All plugins are deduplicated by their src path.
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:168]:
```suggestion
   * @note Plugins are also auto-registered from the `~/plugins` directory and these plugins do not need to be listed in `nuxt.config`. All plugins are deduplicated by their src path.
```

File: packages/schema/src/config/app.ts [hunk 3]
```diff
@@ -165,6 +165,10 @@ export default defineUntypedSchema({
    *
    * It can also be an object with `src` and `mode` keys.
    *
+   * @note Plugins are also auto-registered from the `~/plugins` directory and these plugins do not need to be listed in `nuxt.config`. All plugins are deduplicated by their src path.
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:168]:
```suggestion
   * @note Plugins are also auto-registered from the `~/plugins` directory and these plugins do not need to be listed in `nuxt.config` unless you need to customize their order. All plugins are deduplicated by their src path.
```

File: packages/schema/src/config/app.ts [hunk 4]
```diff
@@ -165,6 +165,10 @@ export default defineUntypedSchema({
    *
    * It can also be an object with `src` and `mode` keys.
    *
+   * @note Plugins are also auto-registered from the `~/plugins` directory and these plugins do not need to be listed in `nuxt.config` unless you need to customize their order. All plugins are deduplicated by their src path.
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:168]:
```suggestion
   * @note Plugins are also auto-registered from the `~/plugins` directory
   * and these plugins do not need to be listed in `nuxt.config` unless you
   * need to customize their order. All plugins are deduplicated by their src path.
```

**Review by pi0 — APPROVED** [review]
Thanks <3

**Pull request merged**